src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncUnorderedSpec.cs

```diff
@@ -193,7 +193,7 @@ await this.AssertAllStagesStoppedAsync(async() => {
                     .RunWith(Sink.FromSubscriber(c), Materializer);
                 var sub = await c.ExpectSubscriptionAsync();
                 sub.Request(10);
-                c.ExpectError().Message.Should().Be("err2");
+                (await c.ExpectErrorAsync()).Message.Should().Be("err2");
```

Comment:
We no longer get `AggregateException`s here unless the test completes immediately, in which case we might have to harden this in the future.